src/org/zaproxy/zap/extension/jxbrowserlinux32/ZapAddOn.xml

```diff
@@ -0,0 +1,29 @@
+<zapaddon>
+	<name>JxBrowser (Linux 32)</name>
+	<version>1</version>
+	<status>alpha</status>
+	<description>An embedded browser based on Chromium, Linux 32 specific </description>
+	<author>ZAP Dev Team</author>
+	<url></url>
+	<changes>First version</changes>
+	<dependencies>
+		<addons>
+			<addon>
+				<id>jxbrowser</id>
+			</addon>
+			<addon>
+				<id>webdriverlinux</id>
+			</addon>
+		</addons>
+	</dependencies>
+	<extensions>
+	    <extension>org.zaproxy.zap.extension.jxbrowserlinux32.ExtensionJxBrowserLinux32</extension>
+	</extensions>
+	<ascanrules/>
+	<pscanrules/>
+	<filters/>
+	<files/>
+	<not-before-version>2.4.0</not-before-version>
```

Comment:
This could be bumped to 2.5.0 as well? (Same comment for other ZapAddOn.xml files.)